vendor/engines/secure_rooms/app/controllers/secure_rooms_api/scans_controller.rb

```diff
@@ -10,19 +10,38 @@ class SecureRoomsApi::ScansController < ApplicationController
   before_action :load_models
 
   def scan
-    response_json = {
-      response: "deny",
-      reason: "I only know how to deny right now.",
-    }
-
-    render json: response_json, status: :forbidden
+    user_accounts = @user.accounts_for_product(@card_reader.secure_room)
```

Comment:
totally agreed, but this is one of those things that seems "helpful" here and "obviously necessary" for another working branch I have already so if we don't add it right now I don't think there's a risk of forgetting it needs to be done